masterlist.yaml

```
@@ -17642,6 +17642,11 @@ plugins:
         name: 'Carriage and Ferry Travel Overhaul'
     after: [ 'Immersive Citizens - AI Overhaul.esp' ]
     msg:
+      - <<: *alsoUseX
+        subs: [ '[Carriage and Ferry Travel Overhaul - Fixes and Winterhold](https://www.nexusmods.com/skyrimspecialedition/mods/40651/)' ]
+      - <<: *alsoUseX
+        subs: [ '[Carriage and Ferry Travel Overhaul - Companions Radiant Quest Fix](https://www.nexusmods.com/skyrimspecialedition/mods/52353/)' ]
+        condition: 'active("CFTO.esp") and not active("CFTO_fix.esp")'
```

1. These should be swapped to be sorted alphabetically by `subs` content.
2. The active check for `CFTO.esp` isn't really necessary in this context.
3. You can make the unconditional message conditional by using the version information of the plugins.

```suggestion
      - <<: *alsoUseX
        subs: [ '[Carriage and Ferry Travel Overhaul - Companions Radiant Quest Fix](https://www.nexusmods.com/skyrimspecialedition/mods/52353/)' ]
        condition: 'not active("CFTO_fix.esp")'
      - <<: *alsoUseX
        subs: [ '[Carriage and Ferry Travel Overhaul - Fixes and Winterhold](https://www.nexusmods.com/skyrimspecialedition/mods/40651/)' ]
        condition: 'version("CFTO.esp", "2.0", <=)'
```